modules/manage/pages/recovery-mode.adoc

```diff
@@ -0,0 +1,218 @@
+= Recovery Mode
+:description: Recovery mode starts Redpanda in "metadata only" mode and disables partitions so you can repair a failed cluster.
```

Comment:
Do we need to explain metadata-only mode? Or, should we just omit it?

Reply:
@JakeSCahill Is something like "...starts Redpanda with limited functionality and disables partitions..." preferable?